src/chips/LightChip.js

```diff
@@ -0,0 +1,43 @@
+import {Helper} from "../Helper";
+
+class LightChip {
+  #areaIds;
+  #options = {
+    // No default options.
+  };
+
+  constructor(areaIds, options = {}) {
+    if (!Helper.isInitialized()) {
+      throw new Error("The Helper module must be initialized before using this one.");
+    }
+
+    this.#areaIds = areaIds;
+    this.#options = {
+      ...this.#options,
+      ...options,
+    };
+  }
+
+  getChip() {
+    return {
+      type: "template",
+      icon: "mdi:lightbulb-group",
+      icon_color: "amber",
+      content: Helper.getCountTemplate("light", "eq", "on"),
+      tap_action: {
+        action: "call-service",
+        service: "light.turn_off",
+        target: {
+          area_id: this.#areaIds,
```

Comment:
Getting this error when tapping
![image](https://github.com/AalianKhan/mushroom-strategy/assets/39784853/e6f1b69c-e4e8-4919-a9e0-4803913058b9)
I think it expects a list. 

The same is with all other chips

Reply:
Fixed in next commit